// school-info/dist/index.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.timetable = exports.search = exports.schedule = exports.meal = exports.major = exports.classroom = exports.classes = void 0;
var func_1 = require("./func");
Object.defineProperty(exports, "classes", { enumerable: true, get: function () { return func_1.classes; } });
Object.defineProperty(exports, "classroom", { enumerable: true, get: function () { return func_1.classroom; } });
Object.defineProperty(exports, "major", { enumerable: true, get: function () { return func_1.major; } });
Object.defineProperty(exports, "meal", { enumerable: true, get: function () { return func_1.meal; } });
Object.defineProperty(exports, "schedule", { enumerable: true, get: function () { return func_1.schedule; } });
Object.defineProperty(exports, "search", { enumerable: true, get: function () { return func_1.search; } });
Object.defineProperty(exports, "timetable", { enumerable: true, get: function () { return func_1.timetable; } });
