// school-info/dist/func/index.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.timetable = exports.search = exports.schedule = exports.meal = exports.major = exports.classroom = exports.classes = void 0;
const classes_1 = __importDefault(require("./classes"));
exports.classes = classes_1.default;
const classroom_1 = __importDefault(require("./classroom"));
exports.classroom = classroom_1.default;
const major_1 = __importDefault(require("./major"));
exports.major = major_1.default;
const meal_1 = __importDefault(require("./meal"));
exports.meal = meal_1.default;
const schedule_1 = __importDefault(require("./schedule"));
exports.schedule = schedule_1.default;
const search_1 = __importDefault(require("./search"));
exports.search = search_1.default;
const timetable_1 = __importDefault(require("./timetable"));
exports.timetable = timetable_1.default;
